internal-packages/ai/jest.config.js

```diff
@@ -22,5 +29,15 @@ module.exports = {
     '!src/**/__tests__/**',
     '!src/**/__mocks__/**'
   ],
-  setupFilesAfterEnv: ['<rootDir>/src/test-setup.ts']
+  setupFilesAfterEnv: ['<rootDir>/src/test-setup.ts'],
+  // Handle ES modules properly
+  transformIgnorePatterns: [
+    'node_modules/(?!(nanoid|@roast)/)'
+  ],
+  extensionsToTreatAsEsm: ['.ts'],
+  moduleNameMapper: {
+    '^nanoid$': '<rootDir>/../../node_modules/.pnpm/nanoid@3.3.11/node_modules/nanoid/index.cjs'
```

Comment:
Why call out nanoid? Maybe nanoid can be replaced, if its a pain? This also seems like it'll break the moment and we upgrade nanoid